test/internal/kubernetes/utils_create.go

```diff
@@ -293,11 +294,30 @@ func initCNSScenarioVars() (map[CNSScenario]map[corev1.OSName]cnsDetails, error)
 				clusterRoleBindingPath: cnsClusterRoleBindingPath,
 				serviceAccountPath:     cnsServiceAccountPath,
 				initContainerArgs: []string{
-					"deploy", "azure-vnet", "-o", "/opt/cni/bin/azure-vnet", "azure-vnet-telemetry",
-					"-o", "/opt/cni/bin/azure-vnet-telemetry", "azure-vnet-ipam", "-o", "/opt/cni/bin/azure-vnet-ipam",
+					"deploy",
+					"azure-vnet", "-o", "/opt/cni/bin/azure-vnet",
+					"azure-vnet-telemetry", "-o", "/opt/cni/bin/azure-vnet-telemetry",
+					"azure-vnet-ipam", "-o", "/opt/cni/bin/azure-vnet-ipam",
 					"azure-swift.conflist", "-o", "/etc/cni/net.d/10-azure.conflist",
 				},
-				configMapPath: cnsSwiftConfigMapPath,
+				configMapPath: cnsSwiftLinuxConfigMapPath,
+			},
+			corev1.Windows: {
+				daemonsetPath:          cnsWindowsDaemonSetPath,
+				labelSelector:          cnsWindowsLabelSelector,
+				rolePath:               cnsRolePath,
+				roleBindingPath:        cnsRoleBindingPath,
+				clusterRolePath:        cnsClusterRolePath,
+				clusterRoleBindingPath: cnsClusterRoleBindingPath,
+				serviceAccountPath:     cnsServiceAccountPath,
+				initContainerArgs: []string{
+					"deploy",
+					"azure-vnet.exe", "-o", "/k/azurecni/bin/azure-vnet.exe",
+					"azure-vnet-telemetry.exe", "-o", "/k/azurecni/bin/azure-vnet-telemetry.exe",
```

Comment:
Does this work if you don't delete the running the process associated with azure-vnet-telemetry.exe ? Maybe first time on BYO cluster it will but will fail afterwards I suppose.